common/src/main/java/com/microsoft/identity/common/internal/controllers/BaseController.java

```diff
@@ -525,6 +528,13 @@ protected AccountRecord getCachedAccountRecord(
                     );
         }
 
+        if (null == targetAccount && parameters.getOAuth2TokenCache() instanceof MsalOAuth2TokenCache) {
+            targetAccount = getAccountWithFRTIfAvailable(
+                    parameters,
+                    (MsalOAuth2TokenCache)parameters.getOAuth2TokenCache()
```

Comment:
nit: space after cast.

Reply:
Addressed via : https://github.com/AzureAD/microsoft-authentication-library-common-for-android/pull/920/commits/b8f4d0ed5b84c37dd34def53aeada19dce3418c0